### PR TITLE
completed javascript and json

### PR DESCRIPTION
**Pull request by salgarcia666**
sal garcia: completed javascript and json
Screenshot (200)

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  title: 'Outsiders',
+  isbn: '9780140385724',
+  author: 'S.E Hinton'
+}];
+console.log(typeof array, array);
+
+console.log(JSON.stringify(array));
+console.log(typeof JSON.stringify(array));
+
+var stringThree = '{ "id": 36844 }';
```
Comment (Zhiwutian) [inline javascript-and-json/main.js:22]: Make sure to add the name property here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update

**Comment by salgarcia666** [comment]
Screenshot (201)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**